### ISSUE TITLE
issue in YMaps.js when no slots are used

### ISSUE THREAD

**Issue by mSnus** [PNKBizz/vue-yandex-map#387]
line 308 of YMaps.js uses `[this.$slots.default]`, but it could be undefined.

That works fine with dev server, but produces error '**this.$slots.default is not a function**' in production build.

Workaround is to place something dummy inside the component, e.g. this fixed the problem:
```
<yandex-map :settings="ysettings" :coords="[62.140545, 77.430923]" zoom="3" @map-was-initialized="mapLoaded">
		+++<my-component slot="balloon"></my-component>
</yandex-map>
```

But it's better to check the existance of the function before it is called.

**Comment by drev74** [comment]
I confirm this bug for Vue 3 and Vite 2.9.15

**Comment by FragsterAt** [comment]
+1 confirmed/
add this to workaround:
```html
      <yandex-map :settings="settings" :coords="coords" :bounds="bounds">
        <div v-if="false"></div>
      </yandex-map>
```

**Comment by PNKBizz** [comment]
fixed in v0.12.04
But i reccommend to use beta for Vue 3.
